recipe/meta.yaml

```diff
@@ -40,6 +42,7 @@ outputs:
         - libnuma
         - rdma-core  # [linux64]
       run:
+        - rdma-core >={{ rdma_core }}  # [linux64]
```

Comment:
This makes sense to ensure we have the intended pinning. If we implement the idea in issue ( https://github.com/conda-forge/rdma-core-feedstock/issues/12 ), we could drop this too